python/ray/tune/tests/test_trial_scheduler.py

```diff
@@ -502,7 +503,7 @@ def result2(t, rew):
             return dict(time_total_s=t, neg_mean_loss=rew)
 
         sched = HyperBandScheduler(
-            time_attr="time_total_s", reward_attr="neg_mean_loss")
+            time_attr="time_total_s", metric="neg_mean_loss", mode="max")
```

Comment:
can you change this to 
```suggestion
            time_attr="time_total_s", metric="mean_loss", mode="min")
```

Reply:
Sorry, ignore this - let's duplicate it and leave your change as is as a test.